model/operation.h

```diff
@@ -206,8 +206,27 @@ namespace minsky
   class DataOp: public ItemT<DataOp, Operation<minsky::OperationType::data>>
   {
     CLASSDESC_ACCESS(DataOp);
+    friend struct SchemaHelper;
+    string m_description;
   public:
-    string description;
+    DataOp() {description("");}
```

Comment:
The default default constructor would be fine too (bb is updated next time it is used).

Reply:
I have also made this the default constructor.